docs/metrics.md

```diff
@@ -224,7 +232,17 @@ It means that each request to the DAG pipeline will update also the metrics for
 
 ## Metrics implementation for MediaPipe Graphs
 
-For [MediaPipe Graphs](./mediapipe.md) metrics endpoint is not supported.
+For [MediaPipe Graphs](./mediapipe.md) execution there are 4 generic metrics which apply to all graphs:
+
+| Type      | Name  | Description |
+| :---    |    :----   |    :----   |
+| counter      | ovms_requests_accepted | api,interface,method,name | Counts number of requests which ended up pushing MediaPipe packet down the graph stream. For example image frame in vision use cases, LLM prompt in text generation use cases. |
+| counter      | ovms_requests_rejected | api,interface,method,name | Counts errors in MediaPipe packet deserialization phase. For example bad image format in vision use cases. Please note that for V3 API, the LLM request is validated at graph node level meaning that packet deserialization always succeeds. Please refer to specific graph definition and implementation. |
+| counter      | ovms_responses | api,interface,method,name | Useful to track number of packets generated by MediaPipe graph. Keep in mind that single request may trigger production of multiple (or zero) packets, therefore tracking number of responses is complementary to tracking accepted requests. For example tracking streaming partial responses of LLM text generation graphs. |
+| gauge      | ovms_current_graphs | name | Number of graphs currently in-process. For unary communication it is equal to number of currently processing requests (each request initializes separate MediaPipe graph). For streaming communication it is equal to number of active client connections. Each connection is able to reuse the graph and decide when to delete it when the connection is closed. |
+
+Exposing custom metrics in calculator implementations (MediaPipe graph nodes) is not supported yet.
```

Comment:
```suggestion
```
It will be mentioned in the LLM calculator document.

Reply:
I think its worth to have the line here as well